content/docs/cli/_index.md

```diff
@@ -6,9 +6,9 @@ h1: Pulumi CLI overview
 no_on_this_page: true
 meta_image: /images/docs/meta-images/docs-meta.png
 menu:
-  cli:
-    name: Overview
-    weight: 3
+  pulumiiac:
```

Comment:
I don't love `pulumiiac`, but I am following the convention we use elsewhere (`pulumiesc` and `pulumicloud`). The alternative would be to go with `pulumi_iac` (and rename `pulumi_esc` and `pulumi_cloud`) which makes my stomach hurt a little less. Happy to go with what the crowd prefers here!